zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/ServiceTaskTest.java

```diff
@@ -36,20 +33,24 @@ public class ServiceTaskTest {
   @ClassRule public static final EngineRule ENGINE = EngineRule.singlePartition();
   private static final String PROCESS_ID = "process";
   private static final ObjectMapper MAPPER = new ObjectMapper();
-  private static final String RESOURCE_ID = "id";
-  private static final String VERSION_TAG = "1v";
 
   @Rule
   public final RecordingExporterTestWatcher recordingExporterTestWatcher =
       new RecordingExporterTestWatcher();
 
   @Test
-  public void shouldCreateJobWithLinkedResources() throws JsonProcessingException {
-    final var record = createResourceRecord();
-    final MutableResourceState resourceState =
-        ((ProcessingDbState) ENGINE.getProcessingState()).getResourceState();
-    resourceState.storeResourceInResourceColumnFamily(record);
-    resourceState.storeResourceInResourceKeyByResourceIdAndVersionTagColumnFamily(record);
+  public void shouldCreateJobWithLinkedResourcesVersionTagBinding() throws JsonProcessingException {
+    final var rpaScript =
+        """
+          {
+          "type": "default",
+          "id": "Rpa_0w7r08e",
+          "executionPlatform": "Camunda Cloud",
+          "executionPlatformVersion": "8.1.0",
```

Comment:
🔧 As this feature releases with 8.7.0, let's adjust this to avoid confusion about supported versions